tests/components/shelly/test_config_flow.py

```diff
@@ -234,18 +294,23 @@ async def test_zeroconf(hass):
     ):
         result = await hass.config_entries.flow.async_init(
             DOMAIN,
-            data={"host": "1.1.1.1", "name": "shelly1pm-12345"},
+            data=DISCOVERY_INFO,
             context={"source": config_entries.SOURCE_ZEROCONF},
         )
         assert result["type"] == "form"
         assert result["errors"] == {}
-
+        context = next(
+            flow["context"]
+            for flow in hass.config_entries.flow.async_progress()
+            if flow["flow_id"] == result["flow_id"]
+        )
+        assert context["title_placeholders"]["name"] == "shelly1pm-12345"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)